src/test/java/org/openrewrite/staticanalysis/RenameLocalVariablesToCamelCaseTest.java

```diff
@@ -458,4 +462,22 @@ void test() {
         );
     }
 
+    @Test
+    void doNotRenameMethodArguments(){
+        //language=java
+        rewriteRun(
+          java(
+            """
+             @Controller
```

Comment:
Are those anotations needed for the test? They are not imported anyway, thus giving incomplete type attribution... maybe we could remove them for simplicity.

Reply:
I added them to show give a more in-depth example of why this is added. I bet most people here recognise which annotations I mean with this.

I did not think the type attribution was worth adding spring as a dependency though.

Although perhaps I should change it to just have an `@Issue` annotation that explains the why?

Reply:
Note: Soon we will flag these missing type attributions in before sources as errors (unless the before type validation is configured to ignore these).

Reply:
Yes, please, reference this issue from the test too. As Knut says, we have plans to fail tests that do not have proper type attribution in the before step...